SwanCustomEnvironments/swancustomenvironments/templates/customenvs.html

```diff
@@ -236,7 +236,7 @@ <h2 id="modal-title" style="text-align: left;"></h2>
 
                 // Decode the chunk into text and split by newlines
                 // Remove the last element (empty string) for avoiding an extra line break
-                const chunked_lines = decoder.decode(value, { stream: true }).split('\n').pop();
+                const chunked_lines = decoder.decode(value, { stream: true }).split('\n').slice(0, -1);
```

Comment:
```suggestion
                const chunked_lines = decoder.decode(value, { stream: true }).split('\n')[:-1];
```

Reply:
I believe that doesnt work for Javascript